Wireframe/index.html

```diff
@@ -1,13 +1,86 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <meta name="description" content="Get to know what is Git." />
```

Comment:
the trialing slash has no effect here
```suggestion
 <meta name="description" content="Get to know what is Git." >